CONTRIBUTING.md

```diff
@@ -408,3 +409,37 @@ A quick checklist:
   - [ ] submit a PR to https://github.com/rubysec/ruby-advisory-db
 - [ ] update nokogiri.org
 - [ ] bump `lib/nokogiri/version/constant.rb` to a prerelease version like `v1.14.0.dev`
+
+## Fuzzing your gumbo-parser changes
```

Comment:
Can you please move this section to just under "How to run the tests"?

Reply:
done